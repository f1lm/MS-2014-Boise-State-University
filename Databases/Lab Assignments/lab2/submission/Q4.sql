-- 4.  Retrieve the names of students who only took courses from his/her major department.
select FName, LName from student where SSN not in(
select S.SSN from student S
inner join grade_report G on G.SSN = S.SSN
inner join course C on C.CourseNo = G.CourseNo
inner join dept D on D.DeptCode = C.DeptCode
and D.DeptCode not in (select distinct DeptCode from major Maj where Maj.SSN = S.SSN));