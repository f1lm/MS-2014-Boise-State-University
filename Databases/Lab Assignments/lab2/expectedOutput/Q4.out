mysql> source Q4.sql;
+-----------+---------+
| FNAME     | LNAME   |
+-----------+---------+
| John      | Smith   |
| Betty     | Gates   |
| Karen     | Geer    |
| Brad      | Bird    |
| Josephine | Freeman |
| Jason     | Freeman |
+-----------+---------+
6 rows in set (0.00 sec)

mysql> notee;
